README.md

```diff
@@ -358,3 +358,35 @@ The `beaver dlang make` command also uses some utilities that could come handy
 if you want to write a custom build script. Take a look at the `lib/dlang.sh`
 file, you'll find the utility functions with documentation in there.
 
+
+### Auto-convert and release tags for D2
+
+For D1 projects that are compatible with D2, there is a special command that
+allows automatically converting to D2, tagging (using the suffix `+d2`), pushing
+to the repo and creating a GitHub release.
+
+You'll need a GitHub OAuth token to be able to push and create the release, you
+should provide it via the `$GITHUB_OAUTH_TOKEN` environment variable. This
+command also expects the variables `$DIST`, `$TRAVIS_REPO_SLUG` and
+`$TRAVIS_TAG` to be defined.
+
+The recommended way to trigger it in the `.travis.yml` is via a special stage,
+after all the rest of the tests passed, and it should be done only for tags.
+
+For example:
+
+```yml
+env:
+    global:
+        # This should really be added via web or the travis tool, but ENCRYPTED
+        - GITHUB_OAUTH_TOKEN=XXX
+        - DIST=xenial
+jobs:
+    include:
+        - stage: D2 Release
```

Comment:
"Convert to D2 + Release" is more descriptive :)

Reply:
I prefer shorter stage names (and it seems obvious that to release D2 you need to convert it first), but if you strongly prefer the more verbose stage name, I can change it.

Reply:
(also, Travis lower cases the stage name, so it will show "Convert to d2 + Release" (d2 instead of D2), this is why I use "D2" at the beginning :D)

Reply:
Main purpose of stages is to make pipeline more descriptive to anyone not very familiar with the project IMHO. So the more specific, the better.

Reply:
Not here, here the only intention is to run something after all (parallel) tests are done. Is more like a sync point. But if it makes a difference to you I can change it to "D2 auto-converted release". OK?